src/figanos/matplotlib/plot.py

```diff
@@ -1949,12 +1979,11 @@ def taylordiagram(
     transform = PolarAxes.PolarTransform()
 
     # Setup the axis, here we map angles in degrees to angles in radius
-    rlocs = np.array([0, 0.2, 0.4, 0.6, 0.8, 1])
-    rlocs_deg = rlocs * 90
-    tlocs = rlocs_deg * np.pi / 180  # convert degrees to radians
+    # Correlation labels
+    rlocs = np.array([0, 0.2, 0.4, 0.6, 0.7, 0.8, 0.9, 0.95, 0.99, 1])
+    tlocs = np.arccos(rlocs)  # Conversion to polar angles
```

Comment:
(*) this follows more closely the reference https://gist.github.com/ycopin/3342888 cited in the code, and it produces figures like in the litterature (e.g. Cannon 2018 again, but many other places too). It's not related to having normalized taylor diagram